buildenv

```diff
@@ -6,7 +7,7 @@ export ZOPEN_COMP="clang"
 
 export ZOPEN_EXTRA_CFLAGS="-fzos-le-char-mode=ascii -std=gnu11 -mnocsect -fno-short-enums -Wno-error -mzos-target=zosv2r4 -I../src"
 
-export ZOPEN_EXTRA_CPPFLAGS="-D_ZOS -D_OPEN_SYS_SOCK_IPV6 -D_UNIX03_SOURCE -D_UNIX03_THREADS -D_XOPEN_SOURCE_EXTENDED=1 -D_POSIX_C_SOURCE=200112L -D_POSIX_SOURCE -DPOSIX_SIGNALS -D_XOPEN_SOURCE=600 -mzos-target=zosv2r4 -I../src-cpp"
+export ZOPEN_EXTRA_CPPFLAGS="-D_OPEN_SYS_SOCK_IPV6 -D_UNIX03_SOURCE -D_UNIX03_THREADS -D_XOPEN_SOURCE_EXTENDED=1 -D_POSIX_C_SOURCE=200112L -D_POSIX_SOURCE -DPOSIX_SIGNALS -D_XOPEN_SOURCE=600 -mzos-target=zosv2r4 -I../src-cpp"
```

Comment:
You can probably simplify these options since some of them are already in https://github.com/ZOSOpenTools/comp_clangport/blob/main/buildenv#L52. For example, `_XOPEN_SOURCE`.

Reply:
I have pushed the changes, can you please review